### DISCUSSION TITLE
Einheit des Body Elements: "LevelBody"

### DISCUSSION THREAD

**Discussion by tschortschde** [mcuiobroker/docuJarvis#35] [Q&A] [answered]
@mcuiobroker 
Vielen Dank nochmal für die Hilfe.
Durch die Anpassung des BlindLevels in https://github.com/mcuiobroker/docuJarvis/discussions/34#discussioncomment-6419241
sind nun die Einheiten etwas durcheinander ;-)
Wie bekomm ich das wieder so hin, dass die Anzeige von 0% bis 100% dargestellt wird?
![image](https://github.com/mcuiobroker/docuJarvis/assets/11274233/61c8af93-83f9-48c2-be78-7e81030cd00e)

**Comment by mcuiobroker** [comment]
Also das geht ja nicht, man hat ja keinen Zahlenwert? min und max sind ja logische Werte.
Dann muss eins der Zusatzprogramme nutzen, siehe Doku.

**Reply by tschortschde** [reply, marked as answer]
Habe eine Möglichkeit gefunden: 
Wenn mann im Gerät die Komponenten Optionen den "Marker Label (Pro)" anpasst geht es ;-)

![image](https://github.com/mcuiobroker/docuJarvis/assets/11274233/d2b1e068-e06f-45d1-97a4-433c3ba6bbe2)
